src/components/Book.jsx

```diff
@@ -2,13 +2,14 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 // Component imports
-import { removeBook } from '../redux/books/books';
+import { removeBook } from '../redux/books/booksSlice';
 
 function Book(props) {
-  const { title, author, id } = props;
+  // eslint-disable-next-line camelcase
+  const { title, author, item_id } = props;
   const [bookTitle] = useState(title);
   const [bookAuthor] = useState(author);
-  const [bookId] = useState(id);
+  const [bookId] = useState(item_id);
```

Comment:
Good job working on the project requirements/ I noticed you disabled eslint in most of your files in the components directory and in this file as well it is against microverse policy to disable eslint. it would be nice if you enabled eslint because linters help us write clean code. Please do this is all files were eslint was disabled.